README.md

```diff
@@ -10,7 +10,27 @@ Consult each package's README for a description and directions for use.
 
 Clone this repository then, from the root, run the following commands:
 
-* `npm install -g lerna`
-* `lerna bootstrap`
+```
+$ npm install
+$ ./node_modules/.bin/lerna bootstrap
```

Comment:
Maybe add a shell script for this? 
```suggestion
./run-first-time.sh
```

Btw, when providing example command, do not include the '$' as it is an invalid command when copy & pasted as example.

Reply:
Updated, thanks!